examples/functions/capture-tone-of-voice/index.ts

```diff
@@ -0,0 +1,46 @@
+import {createClient} from '@sanity/client'
+import {type DocumentEvent, documentEventHandler, type FunctionContext} from '@sanity/functions'
+
+export const handler = documentEventHandler(
+  async ({context, event}: {context: FunctionContext; event: DocumentEvent}) => {
+    // Client for the "production" dataset
```

Comment:
you don't need to specify this in the comment since this will depend on the config for the end user